zipline/utils/run_algo.py

```diff
@@ -166,12 +171,13 @@ def _run(handle_data,
             bundle_data.adjustment_reader,
         )
 
+        # we register our default loader last, after any loaders from users
+        # have been registered via extensions
+        register_pipeline_loader(USEquityPricing, pipeline_loader)
+
         def choose_loader(column):
```

Comment:
We don't need to create a new function here anymore. `global_pipeline_dispatcher` is already a callable with the right signature.